resources/assets/components/PostFilter/index.js

```diff
@@ -18,9 +18,14 @@ class PostFilter extends React.Component {
           <h4>Post Filter</h4>
           <div className="select">
               <select onChange={(event) => this.change(event)}>
```

Comment:
Looks great! Let's make note for future filtering work that we should create a `<Filter>` component that takes in a array of value/labels` and renders the select list and handles the change event. So as we build out multiple filters, we don't have to hard code values and they can be more flexible.

Reply:
love it! I'll add a `@TODO` comment now.